# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/PresShell.cpp
# Commit: 6591f14bbc71
# Full Hash: 6591f14bbc713f11b6316712d931ecc19a443a5c
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-05 03:52:32
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 06dda9072cb7 -r 6591f14bbc71 layout/base/PresShell.cpp
--- a/layout/base/PresShell.cpp	Mon Oct 04 19:28:49 2021 +0000
+++ b/layout/base/PresShell.cpp	Mon Oct 04 19:46:54 2021 +0000
@@ -138,7 +138,6 @@
 #include "nsIScrollableFrame.h"
 #include "nsITimer.h"
 #ifdef ACCESSIBILITY
-#  include "nsAccessibilityService.h"
 #  include "mozilla/a11y/DocAccessible.h"
 #  ifdef DEBUG
 #    include "mozilla/a11y/Logging.h"
@@ -5462,6 +5461,12 @@
 
   mRenderingStateFlags = aState.mRenderingStateFlags;
   mResolution = aState.mResolution;
+#ifdef ACCESSIBILITY
+  if (nsAccessibilityService* accService =
+          PresShell::GetAccessibilityService()) {
+    accService->NotifyOfResolutionChange(this, GetResolution());
+  }
+#endif
 }
 
 void PresShell::SynthesizeMouseMove(bool aFromScroll) {